src/main/java/org/reaktivity/nukleus/http2/internal/routable/stream/SourceInputStreamFactory.java

```diff
@@ -152,7 +152,7 @@ public MessageHandler newStream()
         private final Int2ObjectHashMap<Http2Stream> http2Streams;
         private int lastPromisedStreamId = 0;
 
-        private final AtomicBuffer slab = new UnsafeBuffer(new byte[4096]);
+        private AtomicBuffer slab;
```

Comment:
Changed the field to MutableDirectBuffer.

I need to keep a slab buffer per HTTP2 stream(unlike in HTTP where a buffer per SourceInputStream) . Will handle it in future PRs.